gulpfile.js

```diff
@@ -0,0 +1,139 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview node module that automate tasks.
+ *
+ * @author barnabasmakonda@gmail.com (Barnabas Makonda)
+ */
+
+var yargs = require('yargs')
```

Comment:
Super!! I am on it
On Wed, Nov 25, 2015 at 8:48 AM Sean Lip <notifications@github.com> wrote:

> In gulpfile.js
> <https://github.com/oppia/oppia/pull/1127#discussion_r45831626>:
>
> > +//
> > +//      http://www.apache.org/licenses/LICENSE-2.0
> > +//
> > +// Unless required by applicable law or agreed to in writing, software
> > +// distributed under the License is distributed on an "AS-IS" BASIS,
> > +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +// See the License for the specific language governing permissions and
> > +// limitations under the License.
> > +
> > +/**
> > + * @fileoverview node module that automate tasks.
> > + *
> > + * @author barnabasmakonda@gmail.com (Barnabas Makonda)
> > + */
> > +
> > +var yargs = require('yargs')
>
> Yes, exactly. And this gives us a clear pattern with which to add
> additional commands (with their own args) in the future.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oppia/oppia/pull/1127/files#r45831626>.
>